common-docs/hugo.toml

```diff
@@ -63,20 +63,24 @@ maxAge="6h"
 getenv = ["^HUGO_CURRICULUM_GITHUB_BEARER_TOKEN$"]
 
 
+theme = "common-theme"
```

Comment:
I actually don't think we need to declare common theme any more  if we're importing the module btw